### DISCUSSION TITLE
Is the subject of the email password-reset.liquid "Password Reset Request" customizable?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#17213] [Archive]
We can put `password-reset.liquid` in `extensions\templates` to customize and it's a nice feature.
But the mail is still sent with "Password Reset Request" as subject even when Directus default language is not English.
Is it possible to personalize it?

**Comment by jperestrelo** [comment]
I'm also looking for a way to do this. @Pachat did you find a way?

**Reply by Pachat** [reply]
Sorry. I did not find a way to do it.

**Comment by vitalijalbu** [comment]
How you guys use liquid with directus??👀 very interesting
